src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,25 @@
+.stars {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  list-style: none;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-size: 13px;
+    background-repeat: no-repeat;
+    background-position: center;
```

Comment:
can be simplify 
```suggestion
    background: url(../images/star.svg) no-repeat center;
```

Reply:
when i use it like this, i have an error
![image](https://github.com/mate-academy/layout_catalog/assets/143347271/7fff0963-ef98-4519-8bdf-fd29de6b709c)